source/features/quick-repo-deletion.tsx

```diff
@@ -23,17 +23,12 @@ function handleToggle(event: delegate.Event<Event, HTMLDetailsElement>): void {
 		'.rgh-open-prs-of-forks', // PRs opened in the source repo
 	]);
 
-	if (hasContent && !confirm('This repo has open issues/PRs, are you sure you want to delete everything?')) {
+	if (hasContent && !confirm('This repository has open issues or pull requests, are you absolutely sure?')) {
```

Comment:
Not an upgrade, I want to preserve "delete everything"

Reply:
This was taken from GitHub's popup title